server/test.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
+
+var request = require("request");
+
+var host = "localhost";
+if(process.env.DOCKER_HOST) {
```

Comment:
If running the tests inside Docker becomes the default, it might allow you to remove this condition.